README.md

```diff
@@ -1 +1,19 @@
-# svccat
\ No newline at end of file
+# svccat
+
+This is a collection of materials intended to educate about the use of [Service Catalog](https://github.com/kubernetes-incubator/service-catalog), a
+Kubernetes extension project that implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker). This allows
+cloud application developers to easily provision web services and connect them to their application.
+
+# Presentation
+
+The [presentation](https://github.com/IBM/svccat/tree/master/presentation) section contains a slide deck covering the use case for OSBAPI, as well
+as an overview of the interface between a cloud platform and an OSB broker. It then covers the details of Service-Catalog, the Kube implementation of OSBAPI.
```

Comment:
Might be good to add something about its intended purpose... e.g. `This is intended to be used at a one hour introductory presentation session, such as a Meetup`  or something like that.